src/cmd/tools/m3ctl/yaml/dbCreate.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+package yaml;
+
+import "github.com/m3db/m3/src/query/generated/proto/admin/database.proto";
+message DatabaseCreateRequestYaml {
+  string Operation = 1;
+  admin.DatabaseCreateRequest Request = 2;
```

Comment:
Proto fields should be of form `foo_bar` (i.e. lower case under score separated)